lib/usecases/log_in_storage_use_case.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_survey/model/login_model.dart';
+import 'package:flutter_survey/usecases/base/base_use_case.dart';
+import 'package:flutter_survey/database/secure_storage.dart';
+import 'package:injectable/injectable.dart';
+
+@Injectable()
+class LogInStorageUseCase extends StorageUseCase<LoginModel> {
```

Comment:
The naming is quite confusing IMO, because we don't actually login-in here

What about `StoreLoginUseCase` ? 💭

Reply:
I found that refresh_token request will return same items as login_request,  so I renamed the class as `AuthTokenStorageUseCase`